Assets/HoloToolkit/Input/Scripts/Utilities/Managers/InputManager.cs

```diff
@@ -72,7 +72,8 @@ public bool IsInputEnabled
         /// <summary>
         /// Should the Unity UI events be fired?
         /// </summary>
-        public bool ShouldSendUnityUiEvents { get { return FocusManager.Instance.UnityUIPointerEvent != null && EventSystem.current != null; } }
+        [Obsolete("Will be removed in a future release.  If you need to know if a specific pointer should send Unity UI Events use FocusManager.Instance.GetSpecificPointerEventData()!=null")]
```

Comment:
Why remove/obsolete this instead of rewriting it to return the new check? It's very similar to what this already returns (a null check on something on `FocusManager`).

Reply:
Because I've updated this so that each pointer has a specific ui pointer event.